src/test/java/com/epam/tlmd/util/DriverSingleton.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.tlmd.util;
+
+import org.openqa.selenium.Platform;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+
+
+public class DriverSingleton {
+
+    private static WebDriver driver;
+    private DriverSingleton(){};
+
+
+    public static WebDriver getDriver()
+    {
+        if (null == driver){
+        	DesiredCapabilities capabilities = Init.setDesiredCapabilities();
+            try {
+            	driver = new RemoteWebDriver(new URL("http://localhost:4444/wd/hub"), capabilities);
```

Comment:
This is unacceptable to store this URL here. There is no another option to run elsewhere  except your local.  Please rework your solution to be able to run on remote.